lldb/source/Host/common/Editline.cpp

```diff
@@ -398,6 +397,20 @@ int Editline::GetLineIndexForLocation(CursorLocation location, int cursor_row) {
   return line;
 }
 
+CursorPosition Editline::GetCursorPosition() {
+  if (!m_editline)
+    return {};
+
+  const LineInfoW *info = el_wline(m_editline);
+  if (!info)
+    return {};
+
+  const size_t editline_cursor_col =
+      (int)((info->cursor - info->buffer) + GetPromptWidth()) + 1;
```

Comment:
You're going to static cast this later, so could you make the type here unsigned to begin with?

Reply:
The static cast is to appease the printf-format specifier. Storing the position as a `size_t` seems more canonical, but I also can't imagine a terminal that exceeds `UINT32_MAX`